readme.md

```diff
@@ -45,10 +45,33 @@ Type: `object`
 
 ##### deep
 
-Type: `boolean`<br>
+Type: `boolean | Function`<br>
 Default: `false`
 
-Recurse nested objects and objects in arrays.
+Recursing behaviour for nested objects and objects in arrays
+
+Deep Value can be boolean:
+
+- If `true`, it recurses all nested objects and objects in arrays.
+
+Or a `DeepKeyFilter` function which has the following properties:
+
+- It has signature `deepKeyFilter(sourceKey: string)`.
```

Comment:
The signature does not have a name.

Reply:
```
- It has signature `mapper(sourceKey, sourceValue, source)`.
```

It was adapted from the mapper description. I think users can be confused with different description styles. We should either name in both cases the signatures or leave them blank, also for the mapper method